doc/cfgfile.rst

```diff
@@ -276,6 +276,10 @@ the line with the key:
      contradicts your ``files`` or ``packages`` option, the files in question
      will be included (you can not exclude a full package/extra directory
      or a single file listed in ``files``).
+   * Exclude patterns are applied uniformly across platforms and can use
+     either Unix-style forward-slash (`/`), or Windows-style back-slash (`\`)
```

Comment:
N.B. inline code in rst uses double backticks. It's possible to configure single backticks to mean the same,  but I don't think I have on this project.

We should also build the docs and check whether the backslash shows up - it might be treated as an escape.